Production_scripts/Lookup-01b-Postcode-tests.R

```diff
@@ -11,13 +11,63 @@
 # postcode lookup file.
 
 ####################################################
+
+# Packages
+library(createslf)
+library(openxlsx)
+
+
+# Read in Data---------------------------------------
+
 # Create new and old dataframes with measures for testing
 new_tests <- produce_slf_postcode_tests(haven::read_sav(get_slf_postcode_path()))
 old_tests <- produce_slf_postcode_tests(haven::read_sav(get_slf_postcode_path(update = previous_update())))
 
-####################################################
-# create tests
+
+# Create tests-------------------------------------------
+
+# Compare new and old outputs
 comparison <- produce_test_comparison(old_tests, new_tests)
 
-# END OF SCRIPT
-####################################################
+
+# Produce Outfile----------------------------------------
+
+# Save test comparisons as an excel workbook
+
+# Load excel workbook
+wb <- loadWorkbook(fs::path(
+  get_slf_dir(), "Tests",
+  paste0(latest_update(), "_tests.xlsx")
+))
+# add a new sheet for tests
+addWorksheet(wb, "source_pc_lookup")
+# write comparison output to new sheet
+writeData(
+  wb,
+  "source_pc_lookup",
+  comparison
+)
+# save output
+saveWorkbook(wb,
+  fs::path(
+    get_slf_dir(), "Tests",
+    paste0(latest_update(), "_tests.xlsx")
+  ),
+  overwrite = TRUE
+)
```

Comment:
We spoke about having this create the workbook if it doesn't yet exist. Also packaging it into a function e.g.

`write_tests_xlsx <- function(comparison_data, name)`